services/ocr/ocr.py

```diff
@@ -1,10 +1,10 @@
 
-import cv2, qrcode, pytesseract, time, ast, requests, datetime
 import numpy as np
+import cv2, qrcode, pytesseract, time, ast, pika, datetime
 
 debug_mode = False
 DEBUG_IMG_SCALE = 0.15
-DATABASE_URL = "http://database-service:5000/api"
+RABBITMQ_HOST = "127.0.0.1"
```

Comment:
To stejný a udělal bych si na to helper funkci/globalní variable kterej to získá. A možná se koukni jak to dělám já na database-service, kde mám třídu Rabbitmq na který je ten connection a takže potom jenom vytvořím svojí `new Rabbitmq()` třídu, a pak jí používám v `UserQueue` třídě tak že jí passnu do constructoru, takže potom dám jenom `this.channel` což vrátí ten channel na komunikaci + tam k tomu mám i helper funkce jako `createQueue`

Reply:
A co jsem koukal na dokumentaci tak doporučovali mít co nejméně channelů